### PR TITLE
WSO-3 Updates to ucgy_gbl_international_analytics.js

### PR DESCRIPTION
**Pull request by SandyVallee**
commented out second ga call to second_tracker.

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js [hunk 1]
```diff
@@ -62,10 +62,16 @@
             }
 
             // Analytics: User accesses country details
+           /* Sandy - Ticket WSO-3 - added an if statement to see if click is comming from a certain ID container box for countries and then
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: Break up the comments to be no longer than 70 characters per line

Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: Also no need to worry about your name and ticket number being in comments if you need to save this space. This information will be recorded by GitHub / Jira.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js [hunk 2]
```diff
@@ -74,8 +80,10 @@
         });
 
         // hide country dropdown
+      var itemCloseing = '';
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: Align this line with the code block.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js [hunk 3]
```diff
@@ -62,10 +62,16 @@
             }
 
             // Analytics: User accesses country details
+           /* Sandy - Ticket WSO-3 - added an if statement to see if click is comming from a certain ID container box for countries and then 
+throw this ga event. Not sure why this whole method it attached to the top menu at all. maybe bigger issue but this works.*/
+  
+          var itemClickedID='#'+countryID;
+          if($(itemClickedID).closest('#nav-country-content').attr('id') =='nav-country-content'){
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: Add a space after equal sign(s) to align with the space you have preceding the equal sign(s).

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js [hunk 4]
```diff
@@ -74,8 +80,10 @@
         });
 
         // hide country dropdown
+      var itemCloseing = '';
         $(this).on('hide.bs.dropdown', function(e) {
             country_name = e.relatedTarget.text;
+            itemCloseing= e.relatedTarget;
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: typo: closing

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js [hunk 5]
```diff
@@ -88,10 +96,13 @@
             }
 
             // Analytics: User closes country details
+         /* Sandy - Ticket WSO-3 */
+          if($(itemCloseing).closest('#nav-country-content').attr('id') =='nav-country-content'){
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: correct typo here too: closing. Also add spacing as noted above.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js (hunk 2 above)
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: Correct typo: closing

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international_analytics.js [hunk 6]
```diff
@@ -12,7 +12,7 @@
       var link_title = $(this).text();
 
       ga('send', 'event', 'Tab', 'Click', link_title);
-      if (!!second_handle) {
+     if (!!second_handle) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international_analytics.js]: Looks like a space was removed here (probably debugging). Please add it back in.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js (hunk 1 above)
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: done

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js (hunk 3 above)
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: done

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js (hunk 2 above)
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: done

Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: fixed

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js (hunk 4 above)
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: fixed

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js (hunk 5 above)
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: done

File: .gitignore [hunk 7]
```diff
@@ -49,3 +49,7 @@ sites/default/local_database.inc
 #########
```
Comment (dpfitzsi) [inline .gitignore]: Looks like the .gitignore file has been changed here too. Can you replace the gitignore file in your branch?

File: sites/default/default.settings.php [hunk 8]
```diff
@@ -1,639 +0,0 @@
-<?php
```
Comment (dpfitzsi) [inline sites/default/default.settings.php]: Seems this file was deleted in your branch.

File: sites/default/settings.php [hunk 9]
```diff
@@ -628,3 +628,11 @@
 require_once(dirname(__FILE__) . '/ucalgary_redirects.inc');
```
Comment (dpfitzsi) [inline sites/default/settings.php]: Also this file was changed by Acquia Dev Desktop. You should replace this with a default one from GitHub.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js [hunk 10]
```diff
@@ -62,10 +62,16 @@
             }
 
             // Analytics: User accesses country details
+           /* click region checked before throwing
+           international ga event*/
+  
+          var itemClickedID ='#'+countryID;
+          if($(itemClickedID).closest('#nav-country-content').attr('id') =='nav-country-content'){
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: Add space on far side of operator.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js [hunk 11]
```diff
@@ -62,10 +62,16 @@
             }
 
             // Analytics: User accesses country details
+           /* click region checked before throwing
+           international ga event*/
+  
+          var itemClickedID ='#'+countryID;
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: Add space on far side of operator.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js [hunk 12]
```diff
@@ -74,8 +80,10 @@
         });
 
         // hide country dropdown
+        var itemClosing = '';
         $(this).on('hide.bs.dropdown', function(e) {
             country_name = e.relatedTarget.text;
+            itemClosing = e.relatedTarget;
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_countries/js/ucgy_gbl_international.js]: What is being captured here? Are we trying to get the ID? Value?

File: .gitignore (hunk 7 above)
Comment (SandyVallee) [inline .gitignore]: done but not committed just yet

**Comment by taeram** [comment]
Closing PR as WSO-3 has been cancelled

**Pull request closed without merging**